webapp/components/Registration/CreateUserAccount.vue

```diff
@@ -154,10 +156,19 @@ export default {
       const { name, password, about } = this.formData
       const { email, nonce } = this
       const termsAndConditionsAgreedVersion = VERSION
+      const language = this.$i18n.locale()
       try {
         await this.$apollo.mutate({
           mutation: SignupVerificationMutation,
-          variables: { name, password, about, email, nonce, termsAndConditionsAgreedVersion },
+          variables: {
+            name,
+            password,
+            about,
+            email,
+            nonce,
+            termsAndConditionsAgreedVersion,
+            language,
```

Comment:
Could `LocaleSwitch` component also update the current user's locale?

Reply:
ich hatte sie schon in der arbeit. kam aber nicht zu guten ergebnissen. wäre schön wenn ich das im folge PR anpassen könnte wenn es passt und ein besserer weg ist :)